features/revert-value.yml

```
@@ -0,0 +1,6 @@
+name: revert
+description: The `revert` keyword can be applied to any CSS property, resetting that property to the previous default, provided by either the user or browser stylesheets. For example, setting `display:revert` on a `<div>` will likely result in `display:block`, the browser default style. This is often confused with `initial` which provides the specified initial value of a property, regardless of the element.
```

This is too long- you'll likely need to pick either the example or the comparison with initial.

```suggestion
description: The `revert` CSS keyword resets any property to its previous default, provided by either the user or browser stylesheets. For example, setting `display:/ revert` on a `<div>` will likely result in `display:block`, the browser default style. This is often confused with `initial`.
```